blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/PipelineNodeGraphVisitor.java

```diff
@@ -524,60 +533,60 @@ protected void accumulatePipelineActions(FlowNode node) {
     @Override
     public List<BluePipelineNode> getPipelineNodes(final Link parent) {
         return this.nodes.stream()
-            .map( n -> new PipelineNodeImpl(n, () ->  parent, run))
-            .collect( Collectors.toList() );
+                         .map(n -> new PipelineNodeImpl(n, () -> parent, run))
+                         .collect(Collectors.toList());
     }
 
     @Override
     public List<BluePipelineStep> getPipelineNodeSteps(final String nodeId, Link parent) {
         FlowExecution execution = run.getExecution();
-        if(execution == null){
+        if (execution == null) {
             logger.debug(String.format("Pipeline %s, runid %s  has null execution", run.getParent().getName(), run.getId()));
             return Collections.emptyList();
         }
         DepthFirstScanner depthFirstScanner = new DepthFirstScanner();
         //If blocked scope, get the end node
         FlowNode n = depthFirstScanner
-                        .findFirstMatch(execution.getCurrentHeads(),
-                            input -> (input!= null
-                                    && input.getId().equals(nodeId)
-                                    && (PipelineNodeUtil.isStage(input) || PipelineNodeUtil.isParallelBranch(input))));
+            .findFirstMatch(execution.getCurrentHeads(),
+                            input -> (input != null
+                                && input.getId().equals(nodeId)
+                                && (PipelineNodeUtil.isStage(input) || PipelineNodeUtil.isParallelBranch(input))));
 
-        if(n == null){ //if no node found or the node is not stage or parallel we return empty steps
+        if (n == null) { //if no node found or the node is not stage or parallel we return empty steps
             return Collections.emptyList();
         }
         PipelineStepVisitor visitor = new PipelineStepVisitor(run, n);
         ForkScanner.visitSimpleChunks(execution.getCurrentHeads(), visitor, new StageChunkFinder());
         return visitor.getSteps()
-            .stream()
-            .map( node -> new PipelineStepImpl(node, parent))
-            .collect( Collectors.toList() );
+                      .stream()
```

Comment:
tabs?

Reply:
*shrug* Something about the way I munge code requires me to auto-format fairly often to put everything back where it should be. We shouldn't be storing source code as text in the first place, it's archaic.